app/settings/surveys/attribute-editor.html

```diff
@@ -3,7 +3,7 @@
       <label translate="app.name">Name</label>
       <input type="text" ng-model="editAttribute.label">
    </div>
-   <div class="form-field" ng-if="editAttribute.input !== 'upload'">
+   <div class="form-field" ng-if="editAttribute.input !== 'upload' && editAttribute.type !== 'title' && editAttribute.type !== 'description'">
```

Comment:
Actually given the 3 new checks.. it seems like you could just wrap all the the form after `label` in 1 large ng-if and it'd be cleaner